integration-tests/jdbc/src/main/java/org/apache/camel/quarkus/component/jdbc/JdbcRoutes.java

```diff
@@ -44,5 +47,20 @@ public void process(Exchange exchange) throws Exception {
                         exchange.getIn().setBody(in);
                     }
                 });
+
+        from("direct://headers-as-parameters")
+                .to("jdbc:camel-ds?useHeadersAsParameters=true");
+
+        from("timer://interval-polling?delay=5000&period=1000")
```

Comment:
Maybe we could gain a bit of delay here.

For instance, could we consume records that would be added in `@PostConstruct` phase ?
And then, we could maybe use `timer` with `delay = 0` and `repeatCount = 1` ?
If this idea work, then maybe we could as well shorten `await()` in test ?

And maybe, it could also be possible to avoid producing a log by using a MockEnpoint like [here](https://github.com/apache/camel-quarkus/blob/main/integration-tests/netty/src/main/java/org/apache/camel/quarkus/component/netty/udp/NettyUdpResource.java#L88).

Reply:
@ldrozdo Do you think this approach could add some value ?

Reply:
@aldettinger sorry for the late response, I had pto. Thank you for your hints, they sound good, I will try to implement them.

Reply:
@aldettinger I've implemented your hints, one thing I had to do differently is using `delay=2000` because with 0 it still sometimes failed.

Reply:
Yes, maybe there is a kind of transaction delay or alike so the record are not visible straight away.
I think it's ok to experiment with this, in case of flakiness, we could maybe wait in @PostConstruct that records are visible.
That's great job. It looks clearer (to me at least) :+1: